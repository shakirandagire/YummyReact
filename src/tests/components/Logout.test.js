import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Logout from '../../components/logout';

describe('<Logout/>', () => {
  const component = shallow(<Logout />);
  it('should render properly', () => {
    expect(component.length).toBe(1);
  });
  it('should render logout', () => {
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});

