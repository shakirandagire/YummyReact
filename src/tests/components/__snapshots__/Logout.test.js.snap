// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logout/> should render logout 1`] = `
<Route
  render={[Function]}
/>
`;
